doc/radosgw/federated-config.rst

```diff
@@ -817,7 +817,7 @@ there is a unified namespace between the two regions.
 
 .. _CRUSH Map: ../../rados/operations/crush-map
 .. _Install Ceph Object Gateway: ../../install/install-ceph-gateway
-.. _Cephx Administration: ../../rados/operations/authentication/#cephx-administration
+.. _Cephx Administration: ../../rados/operations/user-management/#managing-users
```

Comment:
we used to have a paragraph referencing this section
```rst
When accessing the Ceph Storage Cluster, each instance of a Ceph Object Gateway
must provide the Ceph Storage Cluster with a user name and key. We recommend
setting up at least one user name and key for each zone. See `Cephx
Administration`_ for a discussion on adding keyrings and keys.
```
this paragraph is now a part of "Create a Keyring", and the link was removed. maybe we can remove the reference as well?